packages/quantic/force-app/main/default/lwc/quanticSort/e2e/quanticSort.e2e.ts

```diff
@@ -0,0 +1,75 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+
+const newestDateSort = 'date descending';
+const defaultSortLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test;
+  if (useCase.value === useCaseEnum.search) {
+    test = fixtures[useCase.value] as typeof testSearch;
+  } else {
+    test = fixtures[useCase.value] as typeof testInsight;
+  }
+
+  test.describe(`quantic sort  ${useCase.label}`, () => {
+    test.describe(`when changing sort option to Newest`, () => {
+      test('should trigger a new search and log analytics', async ({
+        sort,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await sort.clickSortDropDown();
+        await sort.clickSortButton('Newest');
+        const searchResponse = await searchResponsePromise;
+        const {sortCriteria} = searchResponse.request().postDataJSON();
+        expect(sortCriteria).toBe(newestDateSort);
+        await sort.waitForSortUaAnalytics(newestDateSort);
+      });
+    });
```

Comment:
In my opinion for your test to be robust, you shouldn't rely on "Newest" to be the test option.

Your test should be a bit more agnostic to which Sort options exist.

Example
```suggestion
    test.describe(`when changing sort option`, () => {
      test('should trigger a new search and log analytics', async ({
        sort,
        search,
      }) => {
        const searchResponsePromise = search.waitForSearchResponse();
        await sort.clickSortDropDown();
        const {expectedSortName, expectedSortValue} = await sort.getSortAtPosition(1); // Assuming 0 is the currently selected option, 1 should be the one after that.
        await sort.clickSortButton(1); 
        const searchResponse = await searchResponsePromise;
        const {sortCriteria} = searchResponse.request().postDataJSON();
        expect(sortCriteria).toBe(expectedSortValue);
        await sort.waitForSortUaAnalytics(expectedSortValue);
      });
    });
```

Reply:
no, for e2e test small happy path like this, I don't think we need it too robust, as we also want to be sure the order of 'sort' is the same as what we expect at this moment

Reply:
That's just the thing, I see it as the opposite, if the order of sort changes, we don't want the test to fail.

We're testing that the sort you selected is what actually gets sent in UA and in the search response. 

The order doesn't matter, it could be the 2nd, 3rd, whatever. It just needs to be the same.

It is what we do in our other tests, so please make that small change so our test doesn't break if the ordering of the sort changes. :)